modules/sitemaps/sitemap-builder.php

```diff
@@ -0,0 +1,1295 @@
+<?php
+/**
+ * Build the sitemap tree.
+ *
+ * @package Jetpack
+ * @since 4.5.0
+ * @author Automattic
+ */
+
+require_once dirname( __FILE__ ) . '/sitemap-buffer.php';
+require_once dirname( __FILE__ ) . '/sitemap-librarian.php';
+require_once dirname( __FILE__ ) . '/sitemap-logger.php';
```

Comment:
This file and the class instantiation in line 87 should be done only when WP_DEBUG is defined and set to true.